### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by m-hatch**

### CODE REVIEW

**Comment by m-hatch** [comment]
I would recommend creating a directory structure in src. Like src/components, src/styles, etc.

File: src/actions/actions.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+export function togglePlaying() {
+  return { type: 'TOGGLE_PLAYING' }
+};
+
+export function setPlaying(bool) {
+  return { type: 'SET_PLAYING', bool }
+};
+
+export function setBegining(bool) {
```
Comment (michaelcaterisano) [inline src/actions/actions.js]: Change 'setBegining' to 'setBeginning'

File: src/actions/actions.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  return { type: 'SET_BEGINING', bool }
```
Comment (michaelcaterisano) [inline src/actions/actions.js]: change 'SET_BEGINING' to 'SET_BEGINNING'

File: src/reducers/reducer.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      return Object.assign(
+        {}, state, { isPlaying: !state.isPlaying }
+      );
+
+      case 'SET_PLAYING':
+      return Object.assign(
+        {}, state, { isPlaying: action.bool }
+      );
+
+    case 'SET_BEGINING':
```
Comment (michaelcaterisano) [inline src/reducers/reducer.js]: change 'SET_BEGINING' to 'SET_BEGINNING'

**Comment by m-hatch** [comment]
No problem. Spelling fixed.

**Pull request merged**